index.js

```diff
@@ -1,10 +1,41 @@
 /**
  * The function below has no parameters. Instead, access the arguments from the command line.
- * The first argument passed after the filename should be either "plus" or "minus", which represents the kind of calculation that will be done. Every argument afterwards should be a number.
+ * The first argument passed after the filename should be either "plus" or "minus", which represents 
+ * the kind of calculation that will be done. Every argument afterwards should be a number.
  * Depending on the operation, either add up all of the numbers or subtract all of the numbers, from left to right.
- * @returns {number} The result of either adding all numbers or subtracting all numbers, depending on the arguments added to the command line.
+ * @returns {number} The result of either adding all numbers or subtracting all numbers,
+ * depending on the arguments added to the command line.
  */
-function calculator() {}
+function calculator() {
+    if (process.argv.length == 2) {
```

Comment:
If i run `node run.js 1 2 3`, I get `Invalid operation: 1`. How can you update this conditional to make sure you get the correct error message?